components/members/memberInformation.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { viewMemberModalProps } from "../../types/types";
+
+export const MemberInformation = ({
+  setEditingMember,
+  setModalOpen,
+  member,
+}: viewMemberModalProps) => {
```

Comment:
Component get passed in with four but only has 3 up here.

isOpen is NOT necessary for memberInformation